zcash_client_backend/src/data_api/testing.rs

```diff
@@ -1092,6 +1092,64 @@ where
         )
     }
 
+    /// Invokes [`create_proposed_transaction_pczt`] with the given arguments.
+    ///
+    /// [`create_proposed_transaction_pczt`]: super::wallet::create_proposed_transaction_pczt
+    #[cfg(feature = "pczt")]
+    #[allow(clippy::type_complexity)]
+    pub fn create_proposed_transaction_pczt<InputsErrT, FeeRuleT, ChangeErrT>(
```

Comment:
"transaction_pczt" is redundant; consider renaming to `create_proposed_pczt`.

Reply:
A more accurate name would be `create_proposed_transaction_via_pczt`. The PCZT is the mechanism by which the proposed transaction is being created.

Reply:
```suggestion
    pub fn create_pczt_from_proposal<InputsErrT, FeeRuleT, ChangeErrT>(
```